.github/workflows/ci.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       - name: Setup .NET SDK v6.0.x
         uses: actions/setup-dotnet@v1
         with:
-          dotnet-version: 6.0.x
+          dotnet-version: 6.0.102
```

Comment:
Can you add a comment why we are pinning dotnet version ? You can also include a link to dotnet issue.